### PR TITLE
Bug 1913292: Fix flexible scaling when arbiter enabled

### PR DESCRIPTION
**Pull request by cloudbehl**
Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/storage-and-nodes-step.tsx [hunk 1]
```diff
@@ -112,10 +112,10 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
   }, [cpu, dispatch, memory, nodesCount]);
 
   React.useEffect(() => {
-    if (isFlexibleScalingSupported && !stretchClusterChecked) {
+    if (isFlexibleScalingSupported) {
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/storage-and-nodes-step.tsx:115]:
```suggestion
    if (isFlexibleScalingSupported && !stretchClusterChecked) {
```

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/storage-and-nodes-step.tsx [hunk 2]
```diff
@@ -112,10 +112,10 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
...
       dispatch({
         type: 'setEnableFlexibleScaling',
-        value: isFlexibleScaling(nodesCount, zonesCount),
+        value: !stretchClusterChecked && isFlexibleScaling(nodesCount, zonesCount),
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/storage-and-nodes-step.tsx:118]:
```suggestion
        value: isFlexibleScaling(nodesCount, zonesCount),
```

**Review by afreen23 — CHANGES_REQUESTED** [review]
Make this chnage for internal mode as well

**Comment by afreen23** [comment]
> Make this chnage for internal mode as well

My bad , this looks good to me.

**Comment by cloudbehl** [comment]
Internal doesn't have arbiter so this is not a problem in that case. Once my PR for internal is merged I'll fix it

**Pull request merged**